modules/class.js

```diff
@@ -0,0 +1,122 @@
+// Book Class
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+// Local Storage
+export class Store {
+  static getBooks() {
+    let books;
+    // console.log(localStorage.getItem('books'))
```

Comment:
kindly delete this console.log log comment and all unused or commented code for it makes your code base look untidy. Remember Clean code makes your look like a professional.